scripts/gather-files.sh

```diff
@@ -0,0 +1,5 @@
+rm -rf ~/content
+mkdir ~/content
+cp ~/static/snapshot.html ~/content/index.html
+mkdir ~/content/media/
+cp ./media/* ~/content/media/
```

Comment:
We moven nu niet meer alle bestanden. Enkel wat in `media/` staat en de gegeneerde `snapshot.html` (die dan `index.html` heet)